pkg/arvo/app/chat-hook.hoon

```diff
@@ -13,8 +13,14 @@
       state-1
   ==
 ::
-+$  state-1  [%1 state-base]
++$  state-1  [%1 new-state]
 +$  state-0  [%0 state-base]
++$  new-state
+  $:  synced=(map path ship)
+      invite-created=_|
+      allow-history=(map path ?)
+      loaded-cards=(list card)
+  ==
```

Comment:
Questionable naming here. Using the `state-base` in this type definition can help make clear what's actually changing. (This would also help simplify the new-state construction below.)

```hoon
+$  new-state
  $:  loaded-cards=(list card)
      state-base      
  ==
```

Reply:
u right

Reply:
Done